src/main/kotlin/nl/joelchrist/shorty/client/GoogleCloudClient.kt

```diff
@@ -0,0 +1,24 @@
+package nl.joelchrist.shorty.client
```

Comment:
Ik vind dat deze client in een andere package hoort. Maybe: `nl.joelchrist.shorty.google` ?